test/common.sh

```diff
@@ -25,7 +25,7 @@ function convert_to_tools_ver {
   # Tests in 1.20 and 1.21 with their counterpart version's apiserver.
   "1.20"|"1.21") echo "1.19.2";;
   "1.22") echo "1.22.1";;
-  "1.23") echo "1.23.3";;
+  "1.23") echo "1.23.5";;
```

Comment:
@NikhilSharmaWe 
Sorry, this one will need to keep in a follow-up. That was my mistake . Could we revert?
See that we are facing an issue because we are trying to get kubebuilder-tools-1.23.5-darwin-amd64.tar.gz

Reply:
Yes, I was thinking that too

Reply:
Done